packages/rsu_client.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Rsu_client < Package
+  description 'RSU-Client is a RuneScape Client Launcher written for the now Legacy client and now OldSchool.'
+  homepage 'https://github.com/rsu-client/rsu-client'
+  version '4.3.7'
+  source_url 'https://github.com/rsu-client/rsu-client/archive/v4.3.7.tar.gz'
+  source_sha256 '7cc589de111baad956c0c41a4d7f1b45168886bf623dd3886d07fa3d1dfd604c'
+
+  depends_on 'jdk8'
+  depends_on 'p7zip'
+  depends_on 'wxwidgets'
+  depends_on 'xdg_utils'
+  depends_on 'sommelier'
+
+  def self.patch
+    Dir.chdir 'runescape' do
+      system "for f in \$(grep -r /usr/bin .|egrep -v '^Binary'|egrep -o '^[^:]+');do
+                sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' $f
+              done"
+      system "for f in \$(grep -r /usr/bin .|egrep -v '^Binary'|egrep -o '^[^:]+');do
+                sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' $f
+              done"
```

Comment:
Am I seeing double here? LoL

Reply:
Not quite. First, she replaces `#{CREW_PREFIX}` with `/usr`, and then she replaces `/usr` with `#{CREW_PREFIX}`. No idea why, though.

Reply:
Make sure to backslash and bracket `$f`: `\${f}`

Reply:
If I just replace `/usr` with `/usr/local`, you end up with things like `/usr/local/local`, which I'm fairly sure shouldn't happen.

Reply:
What if you did just this?
```
system "for f in \$(grep -r /usr/bin .|egrep -v '^Binary'|egrep -o '^[^:]+');do
                sed -i 's,/usr/local,#{CREW_PREFIX},g' \${f}
              done"
```

Reply:
That wouldn't turn `/usr/bin` into `/usr/local/bin`.

Reply:
The danger of having these two search and replaces here is the assumption that `#{CREW_PREFIX}` is `/usr/local` and that isn't always the case.  Would this work?
```
system "for f in \$(grep -r /usr/bin .|egrep -v '^Binary'|egrep -o '^[^:]+');do
                sed -i 's,/usr/bin,#{CREW_PREFIX}/bin,g' \${f}
              done"
system "for f in \$(grep -r /usr/bin .|egrep -v '^Binary'|egrep -o '^[^:]+');do
                sed -i 's,/usr/local,#{CREW_PREFIX},g' \${f}
              done"
```

Reply:
Hm... I see what you mean about `/usr/local`.
How about this?
```
system "for f in \$(grep -r /usr/bin .|egrep -v '^Binary'|egrep -o '^[^:]+');do
          sed -i 's,/usr/local,/usr,g' $f
        done"
system "for f in \$(grep -r /usr/bin .|egrep -v '^Binary'|egrep -o '^[^:]+');do
          sed -i 's,/usr,#{CREW_PREFIX},g' $f
        done"
```
This moves everything from `/usr/local` into `/usr`, then from `/usr` to `#{CREW_PREFIX}`.

Reply:
That could work also.

Reply:
It's basically what we already have, just without assuming the file starts out with `#{CREW_PREFIX}` in it rather than `/usr/local`.